components/python/jinja2/Makefile

```diff
@@ -11,11 +11,12 @@
 #
 # Copyright (c) 2014 Alexander Pyhalov.  All rights reserved.
 # Copyright 2015, PALO Richard.
+# Copyright 2017 Gary Mills
 #
 
 include ../../../make-rules/shared-macros.mk
 
-PATH=/usr/bin:/usr/gnu/bin:/usr/sbin
+PATH = $(PATH.ill):/usr/sbin
```

Comment:
can we change this to PATH.illumos, so it reflects the full implementation name? We use "gnu" for GNU implementation, so we should use "illumos", not "ill". Besides, PATH.ill is also a little bit weird as ill also has a different meaning.